src/frontend/components/component-info/component-info.html

```diff
@@ -51,22 +50,20 @@ <h4 class="primary-color col pt1 pb2 m0">
 
     <accordion sectionTitle="State" *ngIf="hasState" [defaultExpanded]="hasState">
       <div section-content>
-        <div class="mxn4">
-          <div [ngSwitch]="loadingState">
-            <template [ngSwitchCase]="ComponentLoadState.Received">
-              <bt-render-state
-                [componentMetadata]="componentMetadata"
-                [metadata]="metadata"
-                [level]="0"
-                [path]="path"
-                [state]="state"
-                (updateValue)="onUpdateProperty($event)">
-              </bt-render-state>
-            </template>
-            <template [ngSwitchCase]="ComponentLoadState.Failed">
-              Failed to load component state
-            </template>
-          </div>
+        <div [ngSwitch]="loadingState">
+          <template [ngSwitchCase]="ComponentLoadState.Received">
+            <bt-render-state
+              [componentMetadata]="componentMetadata"
+              [metadata]="metadata"
+              [level]="0"
```

Comment:
@rajinder-yadav Agreed, it can be improved (remove redundancy), but this code was not added in this PR (the surrounding div was just removed). Let's ship this and make other unrelated changes in another PR (maybe one that corrects all redundant bindings in the app).